tests/unit/core/test_tracking.js

```diff
@@ -48,6 +48,18 @@ describe('Tracking', () => {
     ).toBe(true);
   });
 
+  it('should log OFF when not enabled due to Do Not Track', () => {
+    tracking = new Tracking({
+      _isDoNotTrackEnabled: () => true,
+      _log: {
+        info: sinon.stub(),
+      },
+      trackingEnabled: true,
+      trackingId: 'whatever',
+    });
+    sinon.assert.calledWith(tracking._log.info, sinon.match(/OFF/));
```

Comment:
This doesn't actually test if GA is enabled or not but I don't want to block the patch. I'll file for a fixup.